fake_sentry/fake_sentry.py

```diff
@@ -183,7 +192,6 @@ def get_challenge():
     def check_challenge():
         relay_id = flask_request.json["relay_id"]
         assert relay_id == flask_request.headers["x-sentry-relay-id"]
-        assert relay_id in authenticated_relays
```

Comment:
Since we might run multiple processes now, let's just assume everything is fine.